db/migrate/20180103150147_update_default_edition_start_button_text.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateDefaultEditionStartButtonText < Mongoid::Migration
+  def self.up
+    editions = Edition.where(start_button_text: nil, _type: "TransactionEdition").where(:state.ne => "archived")
+
+    editions.each do |edition|
```

Comment:
Could we just do `editions.update_all(start_button_text: "Start now")` here?

Reply:
Also I wonder can we do `TransactionEdition.where(status_button_text: nil).update_all(start_button_text: "Start now")`?

Reply:
That's much simpler, thanks. 👍 

It's interesting that it seems to skip the `save!` validation if you use `update_all`.